ReferenceError: 'undefined_reference_error_maker' is undefined
   at Anonymous function (*test*message*nexttick_throw.js:*:*)
   at _combinedTickCallback (internal/process/next_tick.js:*:*)
   at _tickCallback (internal/process/next_tick.js:*:*)
   at Module.runMain (module.js:*:*)
   at run (bootstrap_node.js:*:*)
   at startup (bootstrap_node.js:*:*)
   at Anonymous function (bootstrap_node.js:*:*)
